tests/server/test_server_validation.py

```diff
@@ -95,10 +95,7 @@ def test_versioned_base_urls(client, index_client, server: str):
 
     This depends on the routers for each kind of server.
     """
-    try:
-        import simplejson as json
-    except ImportError:
-        import json
+    import json
```

Comment:
We already import json at the start of the file, so I do not see why we need to do this here again.
```suggestion
```